Add Support for Custom Content Types when preparing messages

alexrisch opened the pull request:
Added logic in Conversation to handle content type in prepare method Updated bridged module to handle encrypted messages

Requires iOS Sdk Change: https://github.com/xmtp/xmtp-ios/pull/212
Requires Android Sdk Change: https://github.com/xmtp/xmtp-android/pull/150

Commit:
Add Support for Custom Content Types when preparing messages

Added logic in Conversation to handle content type in prepare method
Updated bridged module to handle encrypted messages

Requires iOS Sdk Change: https://github.com/xmtp/xmtp-ios/pull/212
Requires Android Sdk Change:

nplasterer reviewed (approved):
This implementation looks correct to me. Once iOS and Android land this LGTM. 👍

Commit:
fix: Fixed Support for Custom Content Types in Prepare Message

Bumped versions

Commit:
Add Support for Custom Content Types when preparing messages

Added logic in Conversation to handle content type in prepare method
Updated bridged module to handle encrypted messages

Requires iOS Sdk Change: https://github.com/xmtp/xmtp-ios/pull/212
Requires Android Sdk Change:
